split-webapp/split/src/components/NavList.js

```diff
@@ -3,6 +3,7 @@ import { List, ListItem, ListItemText, Divider } from "@material-ui/core";
 import PaymentIcon from "@material-ui/icons/Payment";
 import PeopleOutlineIcon from "@material-ui/icons/PeopleOutline";
 import "../App.css";
```

Comment:
Is this import still being used? Would it be better to remove it if it is unused?

Reply:
Oops, missed this, thanks !